vimeo-networking/src/main/java/com/vimeo/networking/VimeoService.java

```diff
@@ -133,9 +150,26 @@
      */
     // <editor-fold desc="Editing">
     @PATCH
-    Call<Object> edit(@Header("Authorization") String authHeader, @Url String uri,
+    Call<Object> edit(@Header("Authorization") String authHeader,
```

Comment:
If every single request requires an auth header we should move this an `Interceptor` do add the header there (could be done in another PR).